docs/changelogs/CHANGELOG-2.22.md

```diff
@@ -0,0 +1,481 @@
+
+### Breaking Changes
+
+- Use `registry.k8s.io` instead of `k8s.gcr.io` for Kubernetes upstream images. It might be necessary to update firewall rules or mirror registries accordingly ([#11079](https://github.com/kubermatic/kubermatic/pull/11079))
+- KubeVirt: Manual migration of existing MD required. For more information [see our docs](https://docs.kubermatic.com/kubermatic/v2.22/architecture/supported-providers/kubevirt/kubevirt/#migration-from-kkp-221-to-kkp-222) ([#11430](https://github.com/kubermatic/kubermatic/pull/11430))
+- KubeVirt: remove Flavor handling in favour of Instancetype/Preference. action required: manual update of the MD (refer to [our documentation](https://docs.kubermatic.com/kubermatic/v2.22/architecture/supported-providers/kubevirt/kubevirt/#migration-from-kkp-221-to-kkp-222)) ([#11398](https://github.com/kubermatic/kubermatic/pull/11398))
+- Remove experimental support for Thanos in the Prometheus Helm chart ([#11424](https://github.com/kubermatic/kubermatic/pull/11424))
+- Cloud provider specific configurations are prohibited from seed-scoped default cluster templates; for existing seed-scoped default cluster templates these settings should be removed manually ([#11472](https://github.com/kubermatic/kubermatic/pull/11472))
+- OperatingSystemProfiles and OperatingSystemConfigs have been moved to the user clusters ([#11710](https://github.com/kubermatic/kubermatic/pull/11710))
+- CustomOperatingSystemProfile CRD is introduced for maintaining custom OSPs at the seed level. For more information [see our docs](https://docs.kubermatic.com/kubermatic/v2.22/tutorials-howtos/operating-system-manager/usage/#custom-operatingsystemprofiles) ([#11720](https://github.com/kubermatic/kubermatic/pull/11720))
+    - OSP and OSC resources have been moved to user clusters. KKP will take care of migrating the existing resources
+    - Custom OperatingSystemProfiles should now be created for the kind `CustomOperatingSystemProfile` instead of `OperatingSystemProfile` in the seed namespace
+- Action required on applications: helm has been bumped to version 3.11. Due to a [CVE](https://github.com/helm/helm/security/advisories/GHSA-pwcw-6f5g-gxf8) in the previous version, the helm template function [getHostByName](https://helm.sh/docs/chart_template_guide/function_list/#network-functions has been disabled (function will return an empty string).  If you want to enable this function, you have to set `deployOptions.helm.enableDNS` to true and *verify* this function 'getHostByName' is not being used in a chart to disclose any information you do not want to be passed to DNS servers.(c.f. CVE-2023-25165) ([#11887](https://github.com/kubermatic/kubermatic/pull/11887))
+
+### API Changes
```

Comment:
I know that CRD's are technically our API, but maybe we should rename it to `CRD Changes`?

Reply:
Let's keep it as "API Changes" - e.g. the initial machinedeployment annotation is technically not part of a CRD, so I'd like to keep this section flexible.